src/figuresClasses.ts

```diff
@@ -1,19 +1,83 @@
+/* eslint-disable no-useless-constructor */
+enum Shapes {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+enum Color {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
+}
 export interface Figure {
-
+  shape: Shapes;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape = Shapes.Triangle;
 
-}
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0
+      || this.c >= this.a + this.b) {
+      throw new Error('Not valid arguments');
```

Comment:
Check 3 point in the [checklist](https://github.com/mate-academy/ts_figures_classes/blob/master/checklist.md)